observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
     exporters:
       debug:
         verbosity: detailed # <1>
+        sampling_initial: 5 # <2>
+        sampling_thereafter: 200 # <3>
+        use_internal_logger: true # <4>
     service:
       pipelines:
         traces:
-          exporters: [logging]
+          exporters: [debug]
         metrics:
-          exporters: [logging]
+          exporters: [debug]
 # ...
 ----
-<1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<1> Verbosity of the debug export: `detailed`, `normal`, or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<2> Initial number of messages logged per second. The default value is `2` messages per second.
+<3> Sampling rate after the initial number of messages has been logged. Disabled by default with the default `1` value. Sampling is enabled with values greater than `1`. For more information, see the page for the link:https://pkg.go.dev/go.uber.org/zap/zapcore?utm_source=godoc#NewSampler[`zapcore`] package on the Go Project's website.
```

Comment:
> the initial number of messages
How do I know what this value is?

Reply:
Maybe specify the function that you are using so the use knows s/he is in the right place on the page?
```suggestion
<3> Sampling rate after the initial number of messages has been logged. Disabled by default with the default `1` value. Sampling is enabled with values greater than `1`. For more information, see the link:https://pkg.go.dev/go.uber.org/zap/zapcore?utm_source=godoc#NewSampler[`func NewSampler` ] in the zapcore package on the Go Project's website.
```

Reply:
> How do I know what this value is?
Good question. It's actually the value in the previous callout. I've added it as an appositive to make sure every reader understands. Thank you for raising this point.